lib/utils/ensure-single-browser-tab-only.js

```diff
@@ -36,14 +36,14 @@ if (!foundElectron) {
     }
   });
 
-  window.addEventListener('pagehide', unload);
-  window.addEventListener('beforeunload', unload);
-
+  const eventName = /iPad|iPhone|iPod/.test(navigator.userAgent)
+    ? 'pagehide'
+    : 'beforeunload';
+  window.addEventListener(eventName, unload);
```

Comment:
further, for the sake of making it easier to find this in the code with normal search tools, could we duplicate the hookup here?

```js
if ( 'undefined' !== typeof window.onbeforeunload ) {
	window.addEventListener( 'beforeunload', unload );
} else {
	window.addEventListener( 'pagehide', unload );
}
```

^^^ just a pet peeve of mine where we take expected string contents and hide them inside a variable.

Reply:
done